ts/Extensions/Annotations/Annotations.ts

```diff
@@ -1796,10 +1796,20 @@ chartProto.callbacks.push(function (
 
                             // Annotation not connected to any xAxis -
                             // add new row.
-                            if (xAxisIndex === -1) {
-                                const newRow: any = new Array(event.dataRows[0].length);
+                            let isPie, i;
+                            const series = chart.series;
 
-                                newRow.fill('');
+                            for (i = 0; i < series.length; i++) {
+                                isPie = series[i].is('pie') === false;
+                            }
+
+                            if (isPie && xAxisIndex === -1) {
+                                const n = event.dataRows[0].length,
+                                    newRow: any = new Array(n);
```

Comment:
Can type be `string[]` instead of any?

Reply:
It cannot be used there because not all slots of the created table will be just a type string.

Reply:
Right, just checked: `getDataRows()` returns `Array<Array<(number|string)>>`. Still, it's not any :)

Reply:
Here it will be necessary to rebuild this logic, I will improve it in the new ticket.

Reply:
Okay, thanks!